skfda/exploratory/visualization/_magnitude_shape_plot.py

```diff
@@ -158,14 +163,23 @@ class MagnitudeShapePlot:
 
     """
 
-    def __init__(self, fdatagrid, **kwargs):
+    def __init__(
+        self,
+        fdatagrid: FDataGrid,
+        chart: Union[Figure, Axes, None] = None,
+        *,
+        fig: Optional[Figure] = None,
+        axes: Optional[Sequence[Axes]] = None,
+        **kwargs,
```

Comment:
Are they used?

Reply:
kwargs is used by the DirectionalOutlierDetector